<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#order_name_label , QLabel#order_id_label , #order_status_label{
padding-left: 10px
}</string>
  </property>
  <widget class="QLabel" name="order_id_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 246, 242);
border-radius:20px</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vieworder_btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#vieworder_btn{
background-color: rgb(111, 179, 184);
border-radius:25px
}</string>
   </property>
   <property name="text">
    <string>View Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_name_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 246, 242);
border-radius:20px</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_serial_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_status_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 246, 242);
border-radius:20px</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>order_name_label</zorder>
  <zorder>order_id_label</zorder>
  <zorder>vieworder_btn</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>order_serial_label</zorder>
  <zorder>label_3</zorder>
  <zorder>order_status_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
